libs/state/src/lib/core/utils/safe-pluck.ts

```diff
@@ -1,34 +1,96 @@
 import { isKeyOf } from './typing';
+import { isObject } from 'util';
+
+export function safePluck<T extends object, K1 extends keyof T>(
+  stateObject: T,
+  keys: [K1]
```

Comment:
```suggestion
  keys: K1 | [K1]
```
safePluck can work with just providing a single key as well -> for convenience